server/db/User.js

```diff
@@ -1,5 +1,5 @@
 const { Model } = require('sequelize');
-const office = '[a-z][0-9]{1,2}$';
+const password = require('../lib/password');
```

Comment:
Calling hasher module `password` is not very intuitive :name_badge:

Reply:
The way I did it was making helper.js and exporting hash function, then requiring it in other file with the use of destructuring like so: const { hash } = require('../helper'); 
@kronicker 's idea was to make password.js and have all the password related stuff which I think is overkill since there shouldn't be so many password (only) related stuff.

Reply:
![](https://media.giphy.com/media/3ohz6CjdIBhnCyTxU4/200.gif)
😁

Reply:
I'm not voting for `helpers.js` but let's call it hasher or crypto instead of password?